build/template-install-dependencies.yaml

```diff
@@ -1,7 +1,30 @@
-# template-install-keyvault-secrets.yaml
-# Install all secrets needed from KeyVault onto the build machine.
+#template-install-dependencies.yaml
+
+#install dotnet core
 
 steps:
+- task: UseDotNet@2
+  displayName: 'Use .Net Core SDK 3.1'
+  inputs:
+    version: 3.1.x   
+
+- task: UseDotNet@2
+  displayName: 'Use .Net Core SDK 6'
+  inputs:
+    version: 6.0.x   
+
+- task: UseDotNet@2
+  displayName: 'Use .Net Core SDK 7'
+  inputs:
+    version: 7.0.x   
+
```

Comment:
no .net 8, or is that part of the conditional compilation PR?

Reply:
That's part of the other PR

Reply:
Yep, saw it.